sdk/Lusid.Sdk.Tests/tutorials/Instruments/FxOptionExamples.cs

```diff
@@ -150,5 +152,111 @@ public void FxOptionPortfolioCashFlowsExample(ModelSelection.ModelEnum model, bo
             var fxOption = InstrumentExamples.CreateExampleFxOption(isDeliveryNotCash);
             CallLusidGetPortfolioCashFlowsEndpoint(fxOption, model);
         }
+
+        /// <summary>
+        /// Lifecycle management (i.e. conservation of PV on cashflow events) for FX option
+        /// Cash-settled case (CTVoM model):
+        ///      pv(option) = max(spot - strike, 0) before maturity and
+        ///      pv(cashflow) = max(spot_at_maturity - strike, 0) after maturity
+        /// Physically-settled case (CTVoM model):
+        ///      If option is exercised, pay the strike to obtain the underlying.
+        ///      pv(option) = max(spot - strike, 0) before maturity and
+        ///      pv(cash balance) = max(spot_at_maturity - strike, 0) after maturity (assuming FX rates haven't changed)
+        /// </summary>
+        [LusidFeature("F22-45")]
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney, true)]
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney, false)]
+        public void LifeCycleManagementForFxOption(ModelSelection.ModelEnum model, bool isDeliveryNotCash)
+        {
+            // CREATE option
+            var option = (FxOption) InstrumentExamples.CreateExampleFxOption(isDeliveryNotCash: isDeliveryNotCash);
+
+            // CREATE wide enough window to pick up all cashflows associated to the option
+            var windowStart = option.StartDate.AddMonths(-1);
+            var windowEnd = option.OptionSettlementDate.AddMonths(1);
+
+            // CREATE portfolio and add instrument to the portfolio
+            var scope = Guid.NewGuid().ToString();
+            var (instrumentID, portfolioCode) = CreatePortfolioAndInstrument(scope, option);
+
+            // UPSERT option to portfolio and populating stores with required market data.
+            CreateAndUpsertMarketDataToLusid(scope, model, option);
+
+            // CREATE recipe to price the portfolio with
+            var recipeCode = CreateAndUpsertRecipe(scope, model, windowValuationOnInstrumentStartEnd: true);
```

Comment:
as a general comment this approach is very confusing to a new user, the implication is that you need _all_ the items in the recipe generated by`TestDataUtilities.BuildRecipeRequest`, or put another way, what is the minimum requirement for a recipe to generate the cashflows?

Reply:
I would agree with you but this is the way it's currently set up, unless you want to completely rethink how we do examples?

Reply:
I think we need a specific (minimal) recipe per instrument/use case, for now maybe just do it for this PR and raise a jira to retrofit. Otherwise you may end up field lots of support calls from clients

Reply:
Okay, for the record Market Data also works like this, CreateAndUpsertMarketDataToLusid also sometimes upserts more than the minimal set necessary for the use case.

Let me discuss with team about this and get back to you then

Reply:
@rizwansaeed 
After discussion it seems that we in general agree with you, however the priority for now is to get these examples and instrument coverage in. We'll revisit these later, possibly after a cleanup of recipes and training up the rest of the team on the specifics of how they work.